### PR TITLE
feat: #188 Add tests for sidebar

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        <Sidebar
+          cy={SIDEBAR_CY}
+          trigger={SIDEBAR_TRIGGER}
+          linkList={SIDEBAR_LINKS}
+        />
+      </BrowserRouter>
+      ,
+    );
+  };
+
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:45]: for each

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  it("should render sidebar trigger", () => {
+    mountSidebar();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyTrigger)).should("exist");
+  });
+
+  it("should not render sidebar content by default", () => {
+    mountSidebar();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).should("not.exist");
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:53]: maybe less tests

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    mountSidebar();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyTrigger)).click();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).should("exist");
+  });
+
+  it("should sidebar be closed when click cross", () => {
+    mountSidebar();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyTrigger)).click();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).should("exist");
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).children().last().children("button").click();
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:66]: !

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).should("not.exist");
+  });
+
+  it("should sidebar be closed when click background", () => {
+    mountSidebar();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyTrigger)).click();
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent)).should("exist");
+    cy.get("[class*=\"dialogOverlay\"]").click({force: true});
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:74]: !

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    path: "/",
+    value: "Home",
+  },
+  {
+    path: "/users",
+    value: "Users",
+  },
+];
+
+const SIDEBAR_CLOSE_BUTTON = "[class*=\"closeButton\"]";
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:28]: Why do we need this syntax, why but to use ".closeButton"

Does it possible to add data-cy attribute to close button

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+  });
+
+  it("should render sidebar trigger and should not render sidebar content by default", () => {
+    cy.get(getDataCy(SIDEBAR_CY.dataCyTrigger))
+      .should("exist");
+    cy.get(getDataCy(SIDEBAR_CY.dataCyContent))
+      .should("not.exist");
+  });
+
+  it("should sidebar be opened when click trigger", () => {
```
Comment (tritonJS826) [inline mw-webapp/src/component/sidebar/Sidebar.cy.tsx:53]: Strange description, is it a question?

File: mw-webapp/src/utils/ThemeWorker.ts [hunk 7]
```diff
@@ -98,7 +98,7 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.LIGHT]: "transparent",
   },
   border: {
-    [Theme.DARK]: "none",
+    [Theme.DARK]: "1px var(--primaryBackgroundColor) solid",
```
Comment (tritonJS826) [inline mw-webapp/src/utils/ThemeWorker.ts:101]: Thanks for this) but why background and not border?

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx]: WHat for each. i used it yet

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx (hunk 5 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx:28]: Because we using  modules and it's part of className not full

Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx:28]: It's not possible right now

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx (hunk 6 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx:53]: it's not a question . we describe all tests in this way

Comment (tritonJS826) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx:53]: gramatically it is a question

File: mw-webapp/src/component/sidebar/Sidebar.cy.tsx (hunk 5 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/sidebar/Sidebar.cy.tsx:28]: Thanks, added

**Pull request merged**